packages/hoppscotch-js-sandbox/src/apis/response/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { QuickJSContext, QuickJSHandle } from "quickjs-emscripten"
+import * as E from "fp-ts/Either"
+
+export function marshalObjectToVM(
```

Comment:
What is different between this `marshalObjectToVM` and the `utils` implementation of it.

Also documentation!